linux_kernel_modules/cypwifi/scripts/cywifi.sh

```diff
@@ -46,8 +52,22 @@ cy_wifi_init () {
 cy_wifi_start() {
 	# Let's bring the Cypress chip out of reset
 	if [ "${CF3}" = "mdm9x28" ] ; then
-		# DV4 off expander now
-		echo 1  > /sys/devices/platform/expander.0/wl_reg_on
+		# DV3 vs. DV4 diffs
+          	if [ ${BOARD_REV} == "DV3" ] ; then
+                  	if [ ! -d "/sys/class/gpio/gpio${WL_REG_ON_GPIO}" ] ; then
+				echo ${WL_REG_ON_GPIO} > /sys/class/gpio/export
+				echo out  > /sys/class/gpio/gpio${WL_REG_ON_GPIO}/direction
+			fi
+                	WL_REG_ON_VALUE=`cat /sys/class/gpio/gpio${WL_REG_ON_GPIO}/value`
+                        if [ $WL_REG_ON_VALUE -eq 0 ] ; then
+				echo 1  > /sys/class/gpio/gpio${WL_REG_ON_GPIO}/value
+			fi
+		elif [ ${BOARD_REV} == "1.0" ] ; then
+			echo 1  > /sys/devices/platform/expander.0/wl_reg_on
+		# Assuming things don't change on future board revs, otherwise, change here
```

Comment:
I think we can get rid of this `elif` case for now.  If we ever have something different, we can add the case we need then.